src/components/SourceSearch.js

```diff
@@ -58,13 +58,17 @@ const Search = React.createClass({
   componentWillUnmount() {
     const shortcuts = this.context.shortcuts;
     shortcuts.off(`CmdOrCtrl+${L10N.getStr("sources.search.key")}`,
-      this.toggle);
+     this.toggle);
+    shortcuts.off(`CmdOrCtrl+${L10N.getStr("sources.searchAlt.key")}`,
+     this.toggle);
```

Comment:
This might be overkill, but it could be nice to do

```js
const searchKeys = [L10N.getStr("sources.searchAlt.key"), L10N.getStr("sources.searchkey")]

componentWillUnmount() {
  searchKeys.each(key => shortcuts.off(`CmdOrCtrl+${key}`, this.toggle))
}
```

you're call!